runtime/compiler/runtime/J9CodeCacheManager.cpp

```diff
@@ -76,6 +77,24 @@ J9::CodeCacheManager::initialize(bool useConsolidatedCache, uint32_t numberOfCod
    return self()->OMR::CodeCacheManager::initialize(useConsolidatedCache, numberOfCodeCachesToCreateAtStartup);
    }
 
+bool
+J9::CodeCacheManager::isAvailableMemorySafeForAllocation(size_t &requestedCodeCacheSize)
```

Comment:
Why do we pass `requestedCodeCacheSize` by reference?

Reply:
Fixed this.